readme.md

```diff
@@ -1,88 +1,183 @@
-# FingerprintJS
-
-Work in progress, stay tuned.
+<p align="center">
+  <a href="https://fingerprintjs.com">
+    <img src="resources/logo.svg" alt="FingerprintJS" width="300px" />
+  </a>
+</p>
+<p align="center">
+  Browser fingerprinting solution
+</p>
+<p align="center">
+  <a href="https://github.com/fingerprintjs/fingerprintjs/actions?workflow=Test">
+    <img src="https://github.com/fingerprintjs/fingerprintjs/workflows/Test/badge.svg" alt="Build status">
+  </a>
+  <a href="https://www.npmjs.com/package/@fingerprintjs/fingerprintjs">
+    <img src="https://img.shields.io/npm/dt/fingerprintjs2.svg" alt="Total downloads from NPM">
+  </a>
+  <a href="https://www.npmjs.com/package/@fingerprintjs/fingerprintjs">
+    <img src="https://img.shields.io/npm/v/@fingerprintjs/fingerprintjs.svg" alt="Current NPM version">
+  </a>
+</p>
 
 ## Quick start
 
-### In browser
+### Install from CDN
 
 ```html
 <script>
-  function onFingerprintJSLoad(fp) {
-    fp.get().then(({ visitorId }) => console.log(visitorId));
+  function onFingerprintJSLoad(fpAgent) {
+    // The FingerprintJS agent is ready. Get a visitor identifier when you'd like to.
+    fpAgent.get().then(result => {
+      // This is the visitor identifier:
+      const visitorId = result.visitorId;
+      console.log(visitorId);
+    });
   }
 </script>
 <script
-  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3.0.0-beta.2/dist/fp.min.js"
+  async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
   onload="FingerprintJS.load().then(onFingerprintJSLoad)"
 ></script>
 ```
 
-A simple way to start but not recommended because AdBlock and similar browser extensions often block this URL.
-You should at least upload the script to your server.
+This is a simple way to start but not recommended because AdBlock and other browser extensions can block the script.
+You should at least upload the script file to your server.
 
-### Webpack/Rollup/Browserify
+### Or from NPM and use with Webpack/Rollup/Browserify
 
 ```bash
 npm i @fingerprintjs/fingerprintjs
+# or
+yarn add @fingerprintjs/fingerprintjs
 ```
 
 ```js
-import * as FPJS from '@fingerprintjs/fingerprintjs';
+import * as FingerprintJS from '@fingerprintjs/fingerprintjs';
 
 (async () => {
-  const fpjs = await FPJS.load();
-  const { visitorId } = await fpjs.get();
-  console.log(visitorId)
+  // It's good to call this when the application starts
+  const fpAgent = await FingerprintJS.load();
+
+  // The FingerprintJS agent is ready. Get a visitor identifier when you'd like to.
+  const result = await fpAgent.get();
+
+  // This is the visitor identifier:
+  const visitorId = result.visitorId;
+  console.log(visitorId);
 })();
 ```
 
-## Version policy
+## Pro version
 
-The library doesn't promise same visitor identifier between any versions
-but tries to keep them same as much as reasonable.
+Upgrade to [Pro version](https://fingerprintjs.com) to get 99.5% accuracy of identification.
```

Comment:
Bold 99.5% and make it orange (our orange color)

Reply:
AFAIK markdown doesn’t support text coloring. The only way to color is to put the text as a picture. As the sentence is long, the picture will either be long too or split into lines. In the first case the text will be very small on mobile devices, in the second it will look untidy on desktop.

Also I doubt that making it bold will make it more noticeable. I’d prefer to make this sentence be the section header.

Reply:
ok